extension/gulpfile.babel.js

```diff
@@ -89,17 +89,17 @@ gulp.task('babel', () => {
     'app/scripts.babel/app.js',
     'app/scripts.babel/chromereload.js',
     'app/scripts.babel/background.js'])
-    .pipe($.sourcemaps.init())
+    // .pipe($.sourcemaps.init())
```

Comment:
Yeah AFAIK we only want browserify anyway, because Chrome 52 already knows all the ES2015 stuffs.